packages/wikitext/wikitext.1.0.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "Julien SAGOT <julien.sagot@geneanet.org>"
+authors: "Maxence ABELA <maxence.abela+wikitext@gmail.com>"
+homepage: "https://github.com/geneanet/ocaml-wikitext"
+bug-reports: "https://github.com/geneanet/ocaml-wikitext/issues"
+dev-repo: "git+https://github.com/geneanet/ocaml-wikitext.git"
+synopsis: "Wikitext parser"
+build: [
+  [ make "build" ] {ocaml-native}
+]
+install: [ make "install" ]
+remove: ["ocamlfind" "remove" "wikitext"]
```

Comment:
This is no longer needed since the release of opam2, and it would have required n extra dependency on `ocamlfind` otherwise

Reply:
Can you please upstream the change?

Reply:
Sure. Thanks.